spec/models/concerns/questionable_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.describe Questionable do
+  class TestQuestionableModule
+    include Questionable
+
+    attr_accessor :question_1, :question_2, :question_3
+  end
+
+  let(:hash) {
+    {
+      sections: {
+        some_section: {
+          questions: [
+            { name: 'question_1', type: 'string', answers: [{ value: 'open', relevant: true }]},
+            { name: 'question_2', type: 'string', answers: [{ value: 'yes'}]},
+            { name: 'question_3', type: 'string', answers: [{ value: 'standard' }] }
+          ]
+        }
+      }
+    }
+  }
+  let(:schema) { Schemas::Assessment.new(hash) }
+
+  subject(:assessment) { TestQuestionableModule.new }
+
+  before { allow(assessment).to receive(:schema).and_return(schema) }
+
+  describe '#total_questions_with_relevant_answer' do
+    it 'returns the number of questions answered with relevance' do
+      assessment.question_1 = 'open'
+      assessment.question_2 = 'yes'
```

Comment:
It's not very clear why this is relevant. I know that it is, because I've worked on it, but perhaps this should be a bit more evident in the test file, aka:

```ruby
let(:default_relevant_value) { 'yes' }
```